packages/schematics/angular/service-worker/files/ngsw-config.json

```diff
@@ -6,12 +6,9 @@
     "resources": {
       "files": [
         "/favicon.ico",
-        "/index.html"
-      ],
-      "versionedFiles": [
-        "/*.bundle.css",
-        "/*.bundle.js",
-        "/*.chunk.js"
+        "/index.html",
```

Comment:
It's actually removing `versionedFiles` since it currently does not do anything differently than `files` and the `css`/`js` assets as currently defined are not guaranteed to contain a hash.

Reply:
OK, sorry, misread the diff. Another problem I've been asking myself when fixing my own files: it means now the worker itself will be cached (by itself). Is this OK? Can't this be a problem for updating?

Reply:
I don't believe this will cause the worker to be cached, as the `*.js` should get resolved into individual JS files during build time.

In any case, it won't cause issues if the worker script accidentally ends up being cached by the SW. When the browser fetches the worker script it doesn't go through the SW at all, so that's basically a no-op.

Reply:
Yes you're right, after checking, the worker doesn't end up in the final `ngsw-config.json`.